service/src/main/java/bio/terra/policy/service/pao/graph/ExplainWalker.java

```diff
@@ -170,7 +170,7 @@ public ExplainGraph getExplainGraph() {
   private Pao getPao(UUID objectId) {
     Pao pao = paoMap.get(objectId);
     if (pao == null) {
-      pao = paoDao.getPao(objectId);
+      pao = paoDao.getPao(objectId, true);
```

Comment:
This is the one case where we do want to fetch PAOs that have been deleted since the explain walker needs to be able to walk a graph even if some nodes in the graph have been deleted